src/style.css

```diff
@@ -0,0 +1,363 @@
+
+/*STYLES THE OVERAL LAYOUT*/
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: sans-serif;
+    position: relative;
+}
+
+#main-container {
+    margin: 0 5%;
+    height: 100vh;
+    display: grid;
+    grid-template-columns: 1fr 2fr 1fr;
+}
+
+section {
+    padding: 10px;
+    display: grid;
+    grid-template-rows: 40px 1fr;
+}
+
+footer {
+    font-size: 10px;
+    height: fit-content;
+}
+
+
+
+/*RECURRENT STYLES*/
+
+.rounded-container {
+    background-color: rgb(39, 175, 209);
+    height: 20px;
+    color: white;
+    width: 92%;
+    border-radius: 40px;
+    text-align: center;
+    padding: 6px 0;
+}
+
+
+.profile-pic-container {
+    display: block;
+    width: 40px;
+    height: 40px;
+    border-radius: 40px;
+    border: 1px dotted grey;
+    background-image: url("../assets/img/pexels-aisling-kerr-16904943.jpg");
+    background-size: contain;
+}
+
+.profile-pic-container > img {
+    width: 50px;
+    height: auto;
+    object-fit: cover;
+}
+
+.icon {
+    height: 23px;
+    place-self: center;
+}
+
+h2 {
+    display: inline-block;
+    font-size: 18px;
+    margin-bottom: 5px;
+    font-family: "Tahoma", "Arial", sans-serif;
+    font-weight: bolder;
+    margin-left: 10px;
+ }
+
+ h3 {
+    display: inline-block;
+    font-size: 12px;
+    padding-bottom: 5px;
+    font-family: "Tahoma", "Arial", sans-serif;
+    font-weight: bolder;
+    width:fit-content;
+ }
+
+a {
+    color: rgb(24, 107, 128);
+    font-size: 10px;
+}
+
+p {
+    font-size: 10px;
+}
+
+::-webkit-scrollbar {
+    border: none;
+}
+
+/*STYLES THE LEFT-MENU*/
+
+#clone-icon {
```

Comment:
I like how youve switched to use class ID selectors in your left-nav menu

Reply:
cheers! trying to make my css less confusing than it used to be - may or may not have managed to confuse myself rather thoroughly earlier this week ^^